src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.zh-Hant.xlf

```diff
@@ -154,7 +154,7 @@
       </trans-unit>
       <trans-unit id="InvalidAssemblyVersion">
         <source>MC1005: Invalid AssemblyVersion detected: {0}.</source>
-        <target state="translated">MC1005：偵測到無效的 AssemblyVersion: {0}。</target>
+        <target state="translated">MC1005:偵測到無效的 AssemblyVersion: {0}.</target>
```

Comment:
This punctuation is misalmed and should be Chinese the same time

```suggestion
        <target state="translated">MC1005：偵測到無效的 AssemblyVersion: {0}。</target>
```

Reply:
@lindexi Thank you. Updated the file and am requesting the translators to review the translation memories.